source/Halibut.Tests/Halibut.Tests.csproj

```diff
@@ -51,10 +51,11 @@
     <PackageReference Include="Serilog.Sinks.ColoredConsole" Version="3.0.1" />
   </ItemGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'net48' ">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net4.8' ">
```

Comment:
Is this correct?

Reply:
It's wrong, it should be "net48". Perils of putting down a PR for a week or two then picking it back up again, sorry